chainerx_cc/chainerx/context.cc

```diff
@@ -81,6 +86,9 @@ Backend& Context::GetBackend(const std::string& backend_name) {
                 new cuda::CudaBackend{*this}, context_detail::BackendDeleter{[](gsl::owner<Backend*> ptr) { delete ptr; }}};
 #endif  // CHAINERX_ENABLE_CUDA
     } else {
+#ifdef _WIN32
+        throw BackendError{"Backend is not supported in Windows."};
+#else  //_WIN32
         // Load .so file
```

Comment:
clang-format is complaining on this line since it's interpreted as a continuation of the comment above (with mismatched indentation). I think it's easiest to fix by adding an empty line in between.